source/api/plugins/browser-launch-api.md

```diff
@@ -86,6 +95,12 @@ module.exports = (on, config) => {
 
 ### Changing browser preferences:
```

Comment:
@Bkucera This option should have closed this issue: https://github.com/cypress-io/cypress/issues/3671 But I can't reasonably close this when I can't give them a single example of this working.

Reply:
agreed, lets pick the most requested use-case solved by custom preferences and put it as an example here

Reply:
@flotwig would that be setting downloads folder location?

Reply:
@Bkucera yeah, that's a good one if it can be solved via preferences alone

Reply:
@jennifer-shehane  I've made an example changing the download directory here https://github.com/cypress-io/cypress/pull/6494/files#diff-d1e9c5129d8076065e3569141b79b037 I think we can paste that as an example here